filter:
  ros__parameters:
    map_topic: /map
    costmap_clearing_threshold: 70
    info_radius: 1.0
    goals_topic: /detected_frontiers
    global_costmap_topic: /global_costmap/costmap

assigner:
  ros__parameters:
    map_topic: /map
    info_radius: 1.0
    info_multiplier: 3.0
    hysteresis_radius: 3.0
    hysteresis_gain: 2.0
    frontiers_topic: /filtered_goal_points


# # anti_clockwise
# boundry_publisher:
#   ros__parameters:
#     x1y1: 
#     x2y2:
#     x3y3:
#     x4y4:
#     start_coordinates: